src/App.vue

```diff
@@ -229,7 +230,7 @@ export default {
       const breadcrumbs = this.$route?.meta?.breadcrumb
       const crumbs: Array<BreadcrumbIF> = [
         {
-          text: this.getLegalName || this.getCorpTypeNumberedDescription(this.getLegalType),
+          text: this.getLegalName || GetCorpNumberedDescription(this.getLegalType),
```

Comment:
Using function right from shared repo (not locally re-exported) -- as I removed it from Enum Mixin (which I hope to get rid of entirely soon).

Reply:
What do you think of moving `this.getLegalName || GetCorpNumberedDescription(this.getLegalType)` to a store getter?  If I remember correctly, we reuse this logic in the EntityInfo component.   As we've seen with the sole proprietorship discussion, the logic behind what business name to display can get complicated.  I would suggest that any logic like this belongs in a single place where all components can access it.

Reply:
Good idea, will do.

Reply:
Updated in second commit. But there wasn't as much commonality as we thought.